common/src/main/java/dev/ftb/mods/ftblibrary/ui/IntTextBox.java

```diff
@@ -6,7 +6,7 @@
 
 public class IntTextBox extends TextBox {
 
-    private static final Predicate<String> IS_NUMBER = s -> s.matches("^[0-9]+$");
+    private static final Predicate<String> IS_NUMBER = s -> s.matches("^[0-9-]+$");
```

Comment:
`^-?[0-9]+$` would be a better regex here (allow optional '-' at start). Otherwise values like `12-3` would be accepted.